### PR TITLE
Create .zenodo.json

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
This provides the metadata that will be used for the entries on the zenodo repository.

**I ask everyone to fill in their ORCID, affiliation and name if they want to be publicly listed or remove their entry if they don't**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "title": "Plots.jl",
+    "license": "MIT", 
+    "creators": [
+        {
+            "affiliation": "Elemental Cognition", 
+            "name": "Tom Breloff"
+        }, 
+        {
+            "affiliation": "TU Wien", 
+            "name": "Daniel Schwabeneder"
```
Comment (daschw) [inline .zenodo.json:11]:
```suggestion
            "name": "Daniel Schwabeneder"
            "orcid": "0000-0002-0412-0777"
```

**Comment by ChrisRackauckas** [comment]
https://orcid.org/0000-0001-5850-0663, MIT, Chris Rackauckas

**Comment by BoundaryValueProblems** [comment]
Here is my ORCID info: https://orcid.org/0000-0001-5234-4719
Thanks!

**Comment by jheinen** [comment]
https://orcid.org/0000-0001-6509-1925, Forschungszentrum Jülich, Josef Heinen

Thanks!

**Comment by briochemc** [comment]
https://orcid.org/0000-0002-3838-5976, University of Southern California (USC), Benoit Pasquier

😁 Thanks!

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        {
+            "affiliation": "@beacon-biosignals", 
+            "name": "Simon"
+        }, 
+        {
+            "affiliation": "Veos Digital (https://veos.digital/)", 
+            "name": "Pietro Vertechi"
+        }, 
+        {
+            "name": "Zhanibek"
```
Comment (isentropic) [inline .zenodo.json]: name: Zhanibek Omarov
orcid: https://orcid.org/0000-0002-8783-8791

aff: Korea Advanced Inst. of Science and Technology (KAIST)

**Review by isentropic — COMMENTED** [review]
affilitaion

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        {
+            "affiliation": "GLOBE Institute", 
+            "name": "Michael Krabbe Borregaard",
+            "type": "ProjectLeader"
+        }, 
+        {
+            "affiliation": "Leibniz Universit\u00e4t Hannover", 
+            "name": "Simon Christ",
+            "orcid": "0000-0002-5866-1472",
+            "type": "ProjectMember"
```
Comment (daschw) [inline .zenodo.json:26]:
```suggestion
            "type": "ProjectLeader"
```

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        }, 
+        {
+            "affiliation": "Forschungszentrum J\u00fclich", 
+            "name": "Josef Heinen",
+            "orcid": "0000-0001-6509-1925",
+            "type": "ProjectLeader"
```
Comment (daschw) [inline .zenodo.json:32]:
```suggestion
            "type": "ProjectMember"
```

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+        }, 
+        {
+            "name": "Yuval",
+            "type": "Other"
+        }, 
+        {
+            "name": "Andrew Palugniok",
+            "type": "ProjectLeader"
```
Comment (daschw) [inline .zenodo.json:40]:
```suggestion
            "type": "ProjectMember"
```

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+        }, 
+        {
+            "affiliation": "@beacon-biosignals", 
+            "name": "Simon",
+            "type": "Other"
+        }, 
+        {
+            "affiliation": "Veos Digital (https://veos.digital/)", 
+            "name": "Pietro Vertechi",
+            "type": "ProjectLeader"
```
Comment (daschw) [inline .zenodo.json:50]:
```suggestion
            "type": "ProjectMember"
```

**Review by daschw — COMMENTED** [review]
I think you should definitely be classified as ProjectLeader, Simon.

File: .zenodo.json (hunk 3 above)
Comment (BeastyBlacksmith) [reply .zenodo.json:26]: I did the classification based on the share of the current codebase, but I am also fine with your suggestions

File: .zenodo.json [hunk 7]
```diff
@@ -5,79 +5,128 @@
...
         {
             "affiliation": "Veos Digital (https://veos.digital/)", 
-            "name": "Pietro Vertechi"
+            "name": "Pietro Vertechi",
+            "type": "ProjectLeader"
         }, 
         {
-            "name": "Zhanibek"
+            "name": "Zhanibek",
+            "type": "Other"
```
Comment (isentropic) [inline .zenodo.json:54]: name: Zhanibek Omarov
orcid: https://orcid.org/0000-0002-8783-8791

aff: Korea Advanced Inst. of Science and Technology (KAIST)

Comment (BeastyBlacksmith) [reply .zenodo.json:54]: I'd classify you as "ProjectMember" if you don't have objections

Comment (isentropic) [reply .zenodo.json:54]: That is good

**Comment by bernd1995** [comment]
Quick question.
I am about to write a paper and wanted to cite Julia Plots properly.
However I saw that the Zenodo DOI is not regularly updated.
Would it be possible to add a CI that if tags are created also the package on Zenodo is updated? Or should something like that happen already?

**Comment by BeastyBlacksmith** [comment]
For which version do you need a DOI?

**Comment by bernd1995** [comment]
It's not urgent just wanted to let you know 🙂
Paper is still in the final steps and I usually use the latest version of Plots, so whatever version will be around when I am ready to publish 😉

**Comment by kellertuer** [comment]
I think any owner of an org can sync the repos onto his/her zenodo account to activate that per tagged release a DOI is issued. It might be hard to redo that for old releases where this was not active. So if it is activated now a next don might only happen for the next release?

**Comment by BeastyBlacksmith** [comment]
it was already synced, but the .zenodo file was broken and I didn't notice .__.

**Comment by kellertuer** [comment]
Oh, that's sad, because I already had a long discussion with zenodo, when I broke the .zenodo file for one release (and the job running at zenodo did just never finish); and they never wanted to rerun that job. But with a fixed .zenodo it hopefully works on the next release then :)

**Comment by BeastyBlacksmith** [comment]
I could make new point releases for 1.16 - 1.20 to get DOIs for that, but I think its also fine, if people use the general DOI if they can't find it for their version

**Comment by leephillips** [comment]
Lee Phillips
https://orcid.org/0000-0003-4102-2460

**Comment by curio-sitas** [comment]
[0000-0001-8278-394X](https://orcid.org/0000-0001-8278-394X), Université de Rennes, Brian Sinquin

**Comment by BeastyBlacksmith** [comment]
@curio-sitas can you make a PR adding this information?

**Pull request merged**